src/std/format.ss

```diff
@@ -161,6 +162,9 @@
 (defdispatch-q (#\_)
   (write-char #\space))
 
+(defdispatch-q (#\!)
+  (force-output))
```

Comment:
which port is being forced?

Reply:
The current one so it is correct.

Reply:
The default, `(current-output-port)`, that is parameterized by `dofmt`.